tests/sqlparser_common.rs

```diff
@@ -3620,3 +3620,18 @@ fn verified_only_select(query: &str) -> Select {
 fn verified_expr(query: &str) -> Expr {
     all_dialects().verified_expr(query)
 }
+
+#[test]
+fn parse_time_functions() {
```

Comment:
I think we should also test:
1.  the no `()` syntax as well (e. g `SELECT CURRENT_TIMESTAMP`)
2. test coverage for CURRENT_TIMESTAMP, CURRENT_TIME, CURRENT_DATE

Reply:
@alamb I totally agree, the problem is when I try to test the following:

```
let sql = "SELECT CURRENT_TIMESTAMP";
let select = verified_only_select(sql);
```

I got the following error:

```
thread 'parse_time_functions' panicked at 'assertion failed: `(left == right)`
  left: `"SELECT CURRENT_TIMESTAMP"`,
 right: `"SELECT CURRENT_TIMESTAMP()"`', src/test_utils.rs:95:13
```

In the one_statement_parses_to function, The to_string() function of the command returns it with () (Which makes sense, because it's a function without arguments) and I'm not sure how to tackle this one. Would love for some help.

Reply:
perhaps you could use something like `one_statement_parses_to()` instead of `verified_only_select()`

Reply:
@alamb Updated, Thanks!